blueprints/cloud-operations/network-dashboard/main.tf

```diff
@@ -126,7 +128,7 @@ resource "google_cloud_scheduler_job" "job_httptrigger" {
     uri         = module.cloud-function.uri
 
     oidc_token {
-      service_account_email = module.service-account-scheduler.email
+      service_account_email = var.custom_scheduler_sa == "" ? module.service-account-scheduler.email : var.custom_scheduler_sa
```

Comment:
if var.custom_scheduler_sa is provided, this line will never be executed anyway (as the count in this block will be 0). So I think this condition is not required here?